features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/pages/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link href="css/StyleSheet.css" rel="stylesheet" type="text/css">
+    <script src="js/hello.js"></script>
+</head>
+<body>
+<h1>Hello</h1>
+<img src="images/download.jpeg" alt="Hello">
+
+</body>
+</html>
```

Comment:
Add a newline.

Reply:
This was a sample file, this file is no longer in the repo.